### PR TITLE
Set handle keyboard

### PR DESCRIPTION
**Pull request by masatodejima0413**
setHandles move with keyboard!

### CODE REVIEW

File: components/App.tsx [hunk 1]
```diff
@@ -23,6 +25,7 @@ const App = () => {
     setItems(prev => ({ ...prev, [newItem.id]: newItem }));
     newItem.save();
     newSet.save();
+    setFocussedItemId(newItem.id);
   };
```
Comment (masatodejima0413) [inline components/App.tsx:29]: focus item input when new set added

File: components/draggables/DraggableItem.tsx [hunk 2]
```diff
@@ -127,21 +129,27 @@ const DraggableItem = ({
...
     }
 
     if (keyCode === RIGHT_ARROW_KEY_CODE) {
       e.preventDefault();
       itemRefs[itemId].current.focus();
     }
+    if (keyCode === LEFT_ARROW_KEY_CODE) {
+      e.preventDefault();
+      setHandleRef.current.focus();
+    }
```
Comment (masatodejima0413) [inline components/draggables/DraggableItem.tsx:152]: pass ref to setHandle

File: components/draggables/DraggableSet.tsx [hunk 3]
```diff
@@ -72,6 +87,24 @@ const DraggableSet = ({ setId, setIndex, isDraggingOverView }: IProps) => {
...
+      setHandleRefs[prevSetId].current.focus();
+    }
+    if (nextSetId && keyCode === DOWN_ARROW_KEY_CODE) {
+      e.preventDefault();
+      setHandleRefs[nextSetId].current.focus();
+    }
+    if (keyCode === RIGHT_ARROW_KEY_CODE) {
+      e.preventDefault();
+      itemHandleRefs[sets[setId].itemIds[0]].current.focus();
+    }
```
Comment (masatodejima0413) [inline components/draggables/DraggableSet.tsx:105]: pass ref from setHandle to top itemHandle

**Comment by makotodejima** [comment]
Seems working

**Pull request merged**